device/nokia/armhf-nokia_ixs7215_52x-r0/installer.conf

```diff
@@ -1 +1,10 @@
 VAR_LOG_SIZE=4096
+
+if [ -r /host/machine.conf ]; then
+  # fstrim frequency to daily for our platform
+  desc="Discard unused blocks ($platform)"
```

Comment:
@jon-nokia where is 'desc' used?

Reply:
It's used in the sed command to change the description from "Discard unused blocks once a week".  Since I was changing the OnCalendar value the description no longer made sense.